app/frontend/tests/unit/views/form/Create.spec.js

```diff
@@ -0,0 +1,65 @@
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+
+jest.mock('vuex-map-fields', () => ({
+  getterType: jest.fn(),
+  mapFields: jest.fn(),
+}));
+
+import Vuex from 'vuex';
+
+import Create from '@/views/form/Create.vue';
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+
+describe('Create.vue', () => {
+  const mockWindowConfirm = jest.spyOn(window, 'confirm');
+  let store;
+  let formActions = {
```

Comment:
This can be a const - you aren't reassigning the formActions object at any point after the declaration.